src/objects/Error__c.object

```diff
@@ -144,6 +144,29 @@
         <trackTrending>false</trackTrending>
         <type>LongTextArea</type>
         <visibleLines>10</visibleLines>
+    </fields>
+     <fields>
+        <fullName>Record_Id__c</fullName>
+        <description>The Salesforce Id of the record associated with this error.</description>
```

Comment:
Can you have docs review the Label, Description and and InlineHelpText attributes for the two new fields?

Reply:
Already tagged them in the associated QUIP doc.